static/scss/components/buttons.scss

```diff
@@ -41,3 +41,9 @@
     align-items: center;
     padding-left: $spacing-md;
 }
+
+.mzp-c-button.home-sign-in {
+    font-family: $font-metropolis;
+    font-weight: normal;
+    font-size: 1.1rem;
```

Comment:
I overwrote some of the Protocol properties to fit the Figma design. One thing I'm not sure about is increasing the `font-size` to `1.1rem`, because for some reason the button looks subtly smaller by default.

https://user-images.githubusercontent.com/13066134/128549873-38b04aea-bbe4-4ab3-abbd-bdb6afea6bd6.mov

Reply:
Good call out about the `1.1rem`. I would actually recommend updating the `.header-link` size to match the font size of the default button. That keeps it cleaner.  Great job switching to a Protocol button here!